core.py

```diff
@@ -80,16 +106,18 @@ def __init__(self, tof_lib):
         else:
             self.arduino = None
             self.PWMservo = PWM.Servo(pulse_incr_us=1)
-            for pin in range(0, 3):
-                i2c_lidar.xshut([LIDAR_PINS[pin]])
-                self.lidars.append(
-                    i2c_lidar.create(LIDAR_PINS[pin], tof_lib, 0x2a + pin)
-                )
+
+        # I2C lidar sensors should be enabled
+        # in both Arduino mode or direct pi mode.
+        for pin in range(0, 3):
+            i2c_lidar.xshut([LIDAR_PINS[pin]])
```

Comment:
You don't even need that "for pin in LIDAR_PINS": the xshut function takes a list and iterates internally so you can just call it with LIDAR_PINS as the argument, no iteration needed here at all.

These functions have changed in meaning since they were first written and we ditched the XSHUT pin so could do with being renamed/refactored at some point, but as long as the sensors work I can't get worked up about it.